src/main/java/at/ac/fhcampuswien/snake/ingameobjects/Food.java

```diff
@@ -0,0 +1,132 @@
+package at.ac.fhcampuswien.snake.ingameobjects;
+
+import at.ac.fhcampuswien.snake.util.Constants;
+import at.ac.fhcampuswien.snake.util.StateManager;
+
+import java.util.Objects;
+
+public class Food {
+
+    private final Position position;
+
+    private String foodType;
+
+    private boolean isSpecialFood;
+
+    public int getScoreValue() {
+        return scoreValue;
+    }
+
+    private int scoreValue;
```

Comment:
Field may be 'final'
```suggestion
    private final int scoreValue;
```